spec/models/supplejack_api/user_set_spec.rb

```diff
@@ -43,6 +43,12 @@ module SupplejackApi
         expect(user_set.privacy).to eq "public"
       end
 
+      it "sets the subjects field to [] if it is nil" do
+        user_set.subjects = nil
+        user_set.save
+        user_set.reload
+        expect(user_set.subjects).to eq []
```

Comment:
I agree, as I do this in my own work, but all (most of) the tests in sj_api don't currently do this, so I'm sticking with the convention as is in this beast.   Good spotting though